docs/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
```

Comment:
you also renamed this file?

Reply:
yes! this build is currently broken—GitHub pages now expects the main file to be called `index.html`

Reply:
cool ty for fixing!